optuna/samplers/_tpe/sampler.py

```diff
@@ -477,59 +476,75 @@ def _sample(
             self._constraints_func is not None,
         )
 
-        below = self._get_internal_repr(below_trials, search_space)
-        above = self._get_internal_repr(above_trials, search_space)
+        mpe_below = self._build_parzen_estimator(
+            study, search_space, below_trials, handle_below=True
+        )
+        mpe_above = self._build_parzen_estimator(study, search_space, above_trials)
+
+        samples_below = mpe_below.sample(self._rng.rng, self._n_ei_candidates)
+        acq_fn_vals = self._compute_acquisition_function(samples_below, mpe_below, mpe_above)
```

Comment:
The variable name `acq_fn_vals` is not clear. How about `acquisition_function_values`?

Reply:
test